src/index.js

```diff
@@ -189,25 +204,25 @@ export default function undoable (reducer, rawConfig = {}) {
 
   const config = {
     initialState: rawConfig.initialState,
-    initTypes: parseActions(rawConfig.initTypes, ['@@redux/INIT', '@@INIT']),
+    initTypes: parseActions(rawConfig.initTypes, ['@@redux-undo/INIT']),
     limit: rawConfig.limit,
     filter: rawConfig.filter || (() => true),
     undoType: rawConfig.undoType || ActionTypes.UNDO,
     redoType: rawConfig.redoType || ActionTypes.REDO,
     jumpToPastType: rawConfig.jumpToPastType || ActionTypes.JUMP_TO_PAST,
     jumpToFutureType: rawConfig.jumpToFutureType || ActionTypes.JUMP_TO_FUTURE,
+    jumpType: rawConfig.jumpType || ActionTypes.JUMP,
     clearHistoryType: rawConfig.clearHistoryType || ActionTypes.CLEAR_HISTORY
   }
-  config.history = rawConfig.initialHistory || createHistory(config.initialState)
+  config.history = rawConfig.initialHistory || createHistory(config.initialState || reducer(undefined, {}))
```

Comment:
This will set the `config.history` to the initialHistory of the reducer or the passed in config. But when we get the `initialState` from the `(create)Store` it should set it to that right? And there is no way of doing that here because there is not store yet. So I think we should postpone this line to where the reducer is actually called for the first time. I'll try to fix this as well in https://github.com/omnidan/redux-undo/pull/57